wpilibc/src/main/native/include/frc2/command/ParallelCommandGroup.h

```diff
@@ -57,7 +57,12 @@ class ParallelCommandGroup
   // No copy constructors for commandgroups
   ParallelCommandGroup(const ParallelCommandGroup&) = delete;
 
-  template <class... Types>
+  // Prevent template expansion from emulating copy ctor
+  ParallelCommandGroup(SequentialCommandGroup&) = delete;
```

Comment:
Are you sure this isn't a copy paste error?

Reply:
This is not a copy-paste error.  This is the necessary fix, believe it or not.

This is a much less-verbose and cleaner fix than fixing it through SFINAE.